modules/importer/src/server/mod.rs

```diff
@@ -10,14 +10,16 @@ use std::time::Duration;
 use time::OffsetDateTime;
 use tokio::time::MissedTickBehavior;
 use trustify_common::db::Database;
+use trustify_module_storage::service::dispatch::DispatchBackend;
 
 /// run the importer loop
-pub async fn importer(db: Database) -> anyhow::Result<()> {
-    Server { db }.run().await
+pub async fn importer(db: Database, storage: DispatchBackend) -> anyhow::Result<()> {
```

Comment:
Why is a variable named `storage` of a type named `DispatchBackend`?

Reply:
I dunno, honestly. There is a comment left by @ctron in `dispatch.rs` but I didn't fully grok it.